cli/utils.go

```diff
@@ -163,15 +163,15 @@ func setContextIdentity(cmd *cobra.Command, privateKeyHex string) error {
 }
 
 // setContextDocEncryption sets doc encryption for the current command context.
-func setContextDocEncryption(cmd *cobra.Command, shouldEncrypt bool, txn datastore.Txn) {
-	if !shouldEncrypt {
+func setContextDocEncryption(cmd *cobra.Command, shouldEncrypt bool, encryptFields []string, txn datastore.Txn) {
+	if !shouldEncrypt && len(encryptFields) == 0 {
 		return
 	}
 	ctx := cmd.Context()
 	if txn != nil {
 		ctx = encryption.ContextWithStore(ctx, txn)
 	}
-	ctx = encryption.SetContextConfig(ctx, encryption.DocEncConfig{IsEncrypted: true})
+	ctx = encryption.SetContextConfigFromParams(ctx, shouldEncrypt, encryptFields)
```

Comment:
nitpick: Just reading the function signature it can seem odd why `shouldEncrypt` can be false and still have `encryptFields`, but I understand why that is the case due to the two types of options (with field and non-field enc).

Reply:
I can rename it to `shouldEncryptDoc`. Will it make it better?
Otherwise I'm open to alternatives

Reply:
I am okay with that rename :)

Reply:
done